test/env.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+: "${SPLUNK_OPERATOR_IMAGE:=splunk/splunk-operator:latest}"
+: "${SPLUNK_ENTERPRISE_IMAGE:=splunk/splunk:latest}"
+: "${CLUSTER_PROVIDER:=kind}"
+: "${CLUSTER_NAME:=integration-test-cluster}"
+: "${NUM_WORKERS:=3}"
+: "${NUM_NODES:=2}"
+: "${COMMIT_HASH:=}"
+
+# Docker registry to use to push the test images to and pull from in the cluster
+if [ -z "${PRIVATE_REGISTRY}" ]; then
+    case ${CLUSTER_PROVIDER} in
+      kind)
+        PRIVATE_REGISTRY=localhost:5000
+        ;;
+      eks)
+        PRIVATE_REGISTRY=667741767953.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
Think this account info should be provided via CircleCI env (not included in code)

Reply:
Define in ECR_REGISTRY circleci env